content/ngf/how-to/data-plane-configuration.md

```diff
@@ -375,6 +373,36 @@ spec:
 EOF
 ```
 
-For the full configuration API, see the `NginxProxy spec` in the [API reference]({{< ref "/ngf/reference/api.md" >}}).
-
 {{< note >}} When sending curl requests to a server expecting proxy information, use the flag `--haproxy-protocol` to avoid broken header errors. {{< /note >}}
+
+---
+
+## Configure infrastructure-related settings
+
+You can configure deployment and service settings for all data plane instances by editing the `NginxProxy` resource at the GatewayClass level. These settings can also be specified under `nginx.` in the Helm values file. 
+
+Users can specify these settings for NGINX data plane deployments:
+
+  - _replicas_ specifies the number of data plane pod replicas..
```

Comment:
Same note as above, we can maybe mention an example list like "You can edit things such as...", and mention the most important things, but the exhaustive list should just live in the API doc so we don't have to keep updating this document.

And maybe provide an example NginxProxy that sets some of those fields (like replicas, service loadBalancerIP, etc)